components/annot-tab.vue

```diff
@@ -82,6 +82,12 @@
                 const columnList = [];
                 for ( const columnName in this.columnToCategoryMap ) {
 
+                    // If this tab is an assessment tool group, make sure this column is in its toolset
+                    if ( Object.prototype.hasOwnProperty.call(this.details, "groupName") &&
```

Comment:
The `Object.prototype...` thing made me look twice. If I understand right, ESlint flags this because the default `this.details.hasOwnProperty` could fail if `this.details` happens to have a property called `"hasOwnProperty"`? The recommended way to work around this (if legacy browser support isn't a barrier) [seems to be to use `hasOwn` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) instead, which is the replacement of `hasOwnProperty`.

I'd suggest we use `hasOwn` here or if not add a comment why were are using this less expected form.

Reply:
Yup. That's right. `Object.prototype.hasOwnProperty.call` is the suggestion to make sure of the correct behavior. `hasOwn` is newer, considers a few more edge cases, and "is more intuitive." Sounds right.